src/client/_clientwindow.py

```diff
@@ -1152,7 +1152,8 @@ def handle_invalid(self, message):
         raise Exception(message)
 
     def handle_stats(self, message):
-        self.statsInfo.emit(message)
+        # self.statsInfo.emit(message)  # currently useless -> safe some time
```

Comment:
This time safe does not compete with the time lost, when a developer relies on the qt signal and tries to find the error...
If there is no significant slowdown, we should keep it.